meteor/client/ui/Shelf/DashboardPanel.tsx

```diff
@@ -391,26 +391,31 @@ export function getUnfinishedPiecesReactive (rundownId: string, currentPartId: s
 			rundownId: rundownId,
 			partId: currentPartId,
 			dynamicallyInserted: true,
-			$or: [{
-				stoppedPlayback: {
-					$eq: 0
-				}
-			}, {
-				stoppedPlayback: {
-					$exists: false
-				}
-			}, {
-				definitelyEnded: {
-					$exists: false
+			$and: [
+				{
+					$or: [{
+						stoppedPlayback: {
+							$eq: 0
+						}
+					}, {
+						stoppedPlayback: {
+							$exists: false
+						}
+					}],
+				},
+				{
+					definitelyEnded: {
+						$exists: false
+					}
 				}
-			}],
+			],
 			playoutDuration: {
 				$exists: false
 			},
 			adLibSourceId: {
 				$exists: true
 			}
-		}).fetch()
+		}).fetch().filter(p => !(p.userDuration && p.userDuration.duration))
```

Comment:
I couldn't figure out how to query for duration not existing, because userDuration is optional but we shouldn't care whether userDuration exists or not, just whether duration exists. Do you know the right combination of mongo queries?